large-message-core/build.gradle.kts

```diff
@@ -33,12 +33,20 @@ dependencies {
     api(platform("com.bakdata.kafka:kafka-bom:1.1.0"))
     api(group = "org.apache.kafka", name = "kafka-clients")
 
-    implementation(group = "org.slf4j", name = "slf4j-api", version = "2.0.16")
+    val slf4jVersion = "2.0.16"
+    implementation(group = "org.slf4j", name = "slf4j-api", version = slf4jVersion)
+    implementation(group = "org.slf4j", name = "jcl-over-slf4j", version = slf4jVersion)
     val awsVersion = "2.30.18"
-    api(group = "software.amazon.awssdk", name = "s3", version = awsVersion)
-    api(group = "software.amazon.awssdk", name = "sts", version = awsVersion)
+    api(group = "software.amazon.awssdk", name = "s3", version = awsVersion) {
+        exclude(group = "commons-logging", module = "commons-logging")
```

Comment:
Can we move this up for all dependencies?
```
 configurations.all {
    exclude(group = "commons-logging", module = "commons-logging")
}
```

Reply:
The generated pom.xml has an exclusion for every dependency now but I think no one looks at poms and this is a good safeguard if any dependencies are added in the future